tests/Extensions/Compatability.UnitTests/ServiceCollectionExtensionsTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Omex.Extensions.Compatability.Logger;
+using Microsoft.Omex.Extensions.Compatability.TimedScopes;
+using Microsoft.Omex.Extensions.Compatability.Validation;
+using Microsoft.Omex.Extensions.TimedScopes;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Microsoft.Omex.Extensions.Compatability.UnitTests
+{
+	[TestClass]
+	public class ServiceCollectionExtensionsTests
+	{
+		[TestMethod]
+		public void CheckThatTypesRegistred()
+		{
+			Mock<ILogger> mockLogger = new Mock<ILogger>();
+			Mock<ILoggerFactory> mockFactory = new Mock<ILoggerFactory>();
+			mockFactory.Setup(f => f.CreateLogger(It.IsAny<string>())).Returns(mockLogger.Object);
+
+			new HostBuilder()
+				.ConfigureServices(collection =>
+				{
+					collection
+						.AddTimedScopes()
+						.AddSingleton(mockFactory.Object);
+				})
+				.AddOmexCompatabilityServices()
+				.Build()
+				.Start();
+
+			EventId eventId = new EventId(1);
+			Category category = new Category("Test");
+			LogLevel logLevel = LogLevel.Error;
+			string logMessage = "TestLogMessage";
+			Exception exception = new Exception();
+
+			mockLogger.Invocations.Clear();
+			ULSLogging.LogTraceTag(eventId, category, logLevel, logMessage);
+			Assert.AreEqual(1, mockLogger.Invocations.Count, "ULSLogging.LogTraceTag not calling ILogger");
```

Comment:
Keep a single test per test method. It feels like multiple things are being tested here

Reply:
That's true, but since we testing initialization of static types having multiple tests will bring confusion in understanding what code actually initialized them.
I think it might be solved by initializing them with a different object, but that would probably require much more effort.

Reply:
If you use the standard naming format for unit tests, I think that will help to clarify what is going on here. Otherwise, what you're saying makes sense to me.

Reply:
I've updated unit test names for all cases